packages/webpack/example/src/index.js

```diff
@@ -0,0 +1,16 @@
+const Honeybadger = require('@honeybadger-io/js');
+
+Honeybadger.configure({
+  apiKey: (prompt('Enter the API key for your Honeybadger project:')),
```

Comment:
Since `HONEYBADGER_API_KEY` is required to be passed as an env variable for this example app, you could read it from there using the [EnvironmentPlugin](https://webpack.js.org/plugins/environment-plugin/).

Reply:
I do think that would work for our testing purposes, but I don't think it's the example we necessarily want to set for our users. From those docs it looks like this will put the API key into the bundled code as a string, which would expose it publicly. 

I didn't intend this little example to be about the main HB functionality... really just about the sourcemap and deploy upload, but I still wouldn't want to publish something that if copy/pasted could cause user API keys to be public.

Reply:
👍 Gotcha.

Btw, API keys on the client side are exposed publicly anyway in some way or another. Best we can do is hide them under minified/uglified code, but they are exposed nevertheless.